lang/ilias_en.lang

```diff
@@ -6888,6 +6887,7 @@ sahs#:#sahs_sco_objective#:#Objectives
 sahs#:#sahs_sequencing#:#Sequencing
 sahs#:#sahs_unit#:#SCO
 search#:#add_members_header#:#Add Members
+search#:#search_field#:#Search input field
```

Comment:
"Please use uppercase in english (if not a complete sentence): "Search Input Field"